.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Contributing to _WPStarter_
+
+**First**, let us say: Thank you for contributing! :+1:
+
+The outlined steps in this document are just guidelines, not rules, 
+use your best judgment and feel free to propose changes to this document 
+in [a new pull request](https://github.com/wecodemore/wpstarter/compare).
+
+---
+
+## What should I know before getting started?
+
+In general, it's the following rule:
+
+> Try to play nice with later developers or people using this package.
+
+### Submitting changes
+
+1. Please open [an issue](https://github.com/wecodemore/wpstarter/issues/new) 
+   to discuss your changes before filing a pull request.
+2. When you send a pull request, we will love you forever.
+3. Always add changes to **a new branch** named 
```

Comment:
The preview of this, rendered as markdown, doesn't show this as bold. I think only single asterisks are needed.